extensions/transferto/action.go

```diff
@@ -45,8 +45,8 @@ func NewTransferAirtimeAction(uuid flows.ActionUUID, amounts map[string]decimal.
 	}
 }
 
-// Validate validates our action is valid and has all the assets it needs
-func (a *TransferAirtimeAction) Validate(assets flows.SessionAssets, context *flows.ValidationContext) error {
+// Validate validates our action is valid
+func (a *TransferAirtimeAction) Validate() error {
```

Comment:
actions now only do structural validation on themselves - they're not responsible for determining if their dependencies exist - we do that centrally